.ci_scripts/beforeInstall.sh

```diff
@@ -84,14 +90,14 @@ else
     echo "  maxIncomingConnections: $MAX_CON" >> /tmp/mongod.conf
 fi
 
-mkdir /tmp/mongodb
+mkdir -p /tmp/mongodb
```

Comment:
-p stops the command failing if the dir exists, as said useful when re-running the script in the docker container, anyway it's gone now